dashboard/app/controllers/user_school_infos_controller.rb

```diff
@@ -0,0 +1,63 @@
+class UserSchoolInfosController < ApplicationController
+  #before_action :authenticate_user!
```

Comment:
this should _definitely_ be included

Reply:
also, note that if you use `load_resource` or `load_and_authorize_resource` here, you don't need to manually do `user_school_info = UserSchoolInfo.find(params[:id])` in all your route handlers